starlingcaptureapi/file_util.py

```diff
@@ -53,14 +53,35 @@ def digest_sha256(self, file_path):
 
         Returns:
             the HEX-encoded SHA-256 digest of the input file
+
+        Raises:
+            any file I/O errors
         """
         hasher = sha256()
         with open(file_path, "rb") as f:
             # Parse file in blocks
-            for byte_block in iter(lambda: f.read(4096), b""):
+            for byte_block in iter(lambda: f.read(32 * 1024), b""):
+                hasher.update(byte_block)
+            return hasher.hexdigest()
+
+    def digest_md5(self, file_path):
```

Comment:
This function seems identical to `digest_sha256`, except for setting a different `hasher`. I would create a more general function `digest` that takes a parameter to indicate either `sha256` or `md5` algorithm that then picks the correct "hasher".

If we don't want the callers to have to use the more general interface, we can add one-line convenience functions `digest_md5` and `digest_sha256` that call the more general implementation with the correct algorithm.

Reply:
Done in dfb9b23